contracts/VotingUtils.sol

```diff
@@ -352,6 +359,10 @@ contract VotingUtils is EternalStorage, VotingBase {
     return IConsensus(ProxyStorage(getProxyStorage()).getConsensus()).currentValidatorsLength();
   }
 
+  function getStake(address _key) internal returns(uint256) {
```

Comment:
Can add `view` to tihis

Reply:
@leonprou already did this on your branch and have rebase now :)